consensus/spos/roundThreshold_test.go

```diff
@@ -4,7 +4,6 @@ import (
 	"testing"
 
 	"github.com/ElrondNetwork/elrond-go/consensus/spos"
-	"github.com/ElrondNetwork/elrond-go/consensus/spos/bn"
```

Comment:
I think this test should still work with bls

Reply:
done.

Reply:
ok